packages/backend/src/api/endpoints/OrganizationApi.ts

```diff
@@ -5,8 +5,8 @@ import { AbstractAPI } from './AbstractApi';
 
 const basePath = '/organizations';
 
-type MetadataParams = {
-  publicMetadata?: Record<string, unknown>;
+type MetadataParams<TPublic = OrganizationPublicMetadata> = {
```

Comment:
Why do we need the generic here?

Reply:
because `MetadataParams` is used in multiple cases where we have different types of public metadata:
- `UpdateOrganizationMembershipMetadataParams` uses `OrganizationMembershipPublicMetadata` as publicMetadata
- `CreateParams` uses `OrganizationPublicMetadata` as publicMetadata